DirectEmployers/MyJobs
# [PD-2152, PD-2148] Staff-Facing Access Request Form

Preperation
```sh
pip install -r ./requirements.txt --upgrade
```

For Docker users:
```sh
dk rebuilddev
```

# Testing
- visit /request-company-access
  - submit the form and take note of the access code
- visit /admin/myjobs/companyaccessrequest
  - click on the request you just submitted
  - choose a company (that you don't have access to) and enter your access code
- visit / (or refresh)
  - you should see a new company in your drop down list
